scripts/getSnowboundOpenAPISpec

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+npx ts-node 'scripts/getSnowboundOpenAPISpec.ts'
```

Comment:
Why is this a separate shell script? Could we just inline this?

Reply:
🤷  sure